gui/default/assets/css/overrides.css

```diff
@@ -206,6 +206,11 @@ table.table-auto td {
     margin-bottom: 0.25em;
 }
 
+#folderFiles .modal-body {
+    height: 300px;
```

Comment:
There must be a better way of doing this. 

Perhaps we can do something like height 90% (or whatever the margins from the top are to make top and bottom gaps equal)